project/AkkaDisciplinePlugin.scala

```diff
@@ -19,7 +19,28 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
   // We allow warnings in docs to get the 'snippets' right
   val nonFatalWarningsFor = Set("akka-docs")
 
-  val strictProjects = Set("akka-discovery", "akka-protobuf", "akka-coordination")
+  val looseProjects = Set(
+    "akka-actor",
+    "akka-actor-testkit-typed",
+    "akka-actor-tests",
+    "akka-actor-typed",
+    "akka-actor-typed-tests",
+    "akka-bench-jmh",
+    "akka-cluster",
+    "akka-cluster-metrics",
+    "akka-distributed-data",
+    "akka-persistence",
+    "akka-persistence-tck",
+    "akka-persistence-typed",
+    "akka-persistence-query",
+    "akka-remote",
+    "akka-remote-tests",
+    "akka-stream",
+    "akka-stream-testkit",
+    "akka-stream-tests",
+    "akka-stream-tests-tck",
+    "akka-testkit",
+  )
```

Comment:
Makes me sad

Reply:
I don't know what to offer yet but this seems like why bother with the discipline improvements/checks/flags in our build to improve code/keep it clean.

Reply:
I just noticed the flip. Though I agree with @chbatey.